purchase_supplier_approve/__manifest__.py

```diff
@@ -0,0 +1,16 @@
+# Copyright 2019 RGB Consulting - Domantas Sidorenkovas
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+{
+    'name': "Purchase Supplier Approve",
+    'version': '12.0.1.0.0',
+    'category': 'Purchase',
+    'depends': ['purchase'],
+    'author': "RGB Consulting",
```

Comment:
Add OCA as author.

Reply:
Hi,
As you can see I made this change in the following commit:
https://github.com/OCA/purchase-workflow/pull/744/commits/b6f823d3cfa9d4a71fd8cc4b6128cda08aeb6d33#diff-8e65f9724534767cee0c8449ebf1c8dcR9

Reply:
:+1: